WcaOnRails/app/models/role.rb

```diff
@@ -0,0 +1,4 @@
+# frozen_string_literal: true
+
+class Role < ApplicationRecord
+end
```

Comment:
```suggestion
class Role < ApplicationRecord
  belongs_to :user
end
```

Reply:
That way, you can load `my_role = Role.find(123)` and then use `my_role.user` directly. Rails is smart enough to figure out the link as long as the database columns follow certain naming conventions (i.e. using `belongs_to :user` only works if there is a DB column `user_id`. If there wasn't, you would have to tell Rails explicitly how the reference column is called)

Reply:
Cool, thanks. Done.